mflowgen/tile_array/construct.py

```diff
@@ -0,0 +1,245 @@
+#! /usr/bin/env python
+#=========================================================================
+# construct.py
+#=========================================================================
+# Author : 
+# Date   : 
+#
+
+import os
+import sys
+
+from mflowgen.components import Graph, Step
+
+def construct():
+
+  g = Graph()
+
+  #-----------------------------------------------------------------------
+  # Parameters
+  #-----------------------------------------------------------------------
+
+  adk_name = 'tsmc16'
+  adk_view = 'stdview'
+
+  parameters = {
+    'construct_path' : __file__,
+    'design_name'    : 'Interconnect',
+    'clock_period'   : 10.0,
+    'adk'            : adk_name,
+    'adk_view'       : adk_view,
+    # Synthesis
+    'flatten_effort' : 3,
+    'topographical'  : False,
+    # Floorplan
+    'core_width'     : 3000.0,
+    'core_height'    : 2200.0,
```

Comment:
Should the core_width and core_height still be set?

Reply:
Nope. Just removed.